pkg/reader/reader.go

```diff
@@ -1,58 +1,171 @@
-// SPDX-FileCopyrightText: Copyright 2023 The StarBOM Authors
```

Comment:
Don't drop the copyright notices, rather we should rename them to "The Protobom Authors" for now

Reply:
but is it still `// SPDX-FileCopyrightText: Copyright 2023 The StarBOM Authors` but with `// SPDX-FileCopyrightText: Copyright 2023 The Protobom Authors` 

do we want to put it everywhere though? its the only file I've noticed this one